### DISCUSSION TITLE
Help! SQLite Studio wont work on my macbook pro

### DISCUSSION THREAD

**Discussion by bezaabraha** [pawelsalawa/sqlitestudio#5432] [General]
Hi, I need help ASAP!

I downloaded SQLite Studio for an online course. Whenever I try to open or use the software this dialog box always shows up. I'm not sure what I should do. 
<img width="1436" alt="Screen Shot 2024-03-18 at 12 26 38 PM" src="https://github.com/pawelsalawa/sqlitestudio/assets/118004333/c36aea00-8285-44b5-8523-f85f7e4f40d5">

**Comment by tuffnatty** [comment]
@bezaabraha You don't specify your OS version but I am guessing it's a version before 11.0. You can:
1) ignore the message if you don't need SQLiteCipher support;
2) upgrade your OS;
3) use a CI build from https://github.com/pawelsalawa/sqlitestudio/actions/runs/8320673381 , e.g. this one: https://github.com/pawelsalawa/sqlitestudio/actions/runs/8320673381/artifacts/1333849053
